BlazorApp1/wwwroot/blazor-fusionCharts.js

```diff
@@ -1,33 +1,72 @@
+
+const FUNCTION_KEYWORD = 'function()';
+
+// reviver function, key is passed by the JSON.parse
 function parseFunction(key, value) {
-        if (typeof (value) === 'string' && value.includes('function()')) {
-            let regEx = new RegExp("{([^}]*)}");
-            let func = new Function(value.split("function()")[1]);
-            return func;
-        }
-        return value
-    }
-    // Responsible for rendering any chart//
-    window.FusionCharts.renderChart = (chartConfiguration) => {
-        options = JSON.parse(chartConfiguration, this.parseFunction);
-        const chart = new FusionCharts(options);
-        chart.render();
-    };
-    //Generic Method to call any fusion chart method exluding except above methods//
-    window.FusionCharts.invokeChartFunction = (functionName, chartID, ...args) => {
-        var currentChart = FusionCharts(chartID);
-        var result =   currentChart[functionName].apply(currentChart, ...args);
-        var typeofresult = typeof result;
-        if(typeofresult === "number" || typeofresult === "boolean"){
-            result = result.toString();
-        }
-        else if(typeofresult === "object"){
-            result = JSON.stringify(result);
-        }
-        else if(typeofresult === "XML"){
-            result = XML.stringify(result);
-        }
-        if(typeofresult === "string"){
-            return result;
-        }
-        return result.toString() || "";
-    };
\ No newline at end of file
+  if (typeof value === "string" && value.includes(FUNCTION_KEYWORD)) {
+    let functionParts = value.split("function()")
```

Comment:
please use the FUNCTION_KEYWORD variable in the split function

Reply:
done.